Added 3.10.2 changelog (#700)
MapsPeople/docs: src/changelogs/android/v3/CHANGELOG.md

### Security     in case of vulnerabilities.
-->

## [3.10.2] 2021-09-13

## Added

> - MapsIndoors now triggers as ready when locations are loaded.

Reviewer: ```suggestion
- MapsIndoors now triggers as ready when Locations are loaded.
```

> - MapsIndoors now triggers as ready when locations are loaded.

Reviewer: ```suggestion
- MapsIndoors now triggers as ready when Locations are loaded.
```

### Fixed

> - Issue with setApiKey loading new solutions fixed. You can now switch between solutions again by calling setAPIKey, and initialising a new MapControl object.

Reviewer: ```suggestion
- Issue with `setApiKey` loading new Solutions fixed. You can now switch between Solutions again by calling `setAPIKey`, and initializing a new `MapControl` object.
```

> - Issue with setApiKey loading new solutions fixed. You can now switch between solutions again by calling setAPIKey, and initialising a new MapControl object.

Reviewer: ```suggestion
- Issue with `setApiKey` loading new Solutions. You can now switch between Solutions again by calling `setAPIKey`, and initializing a new `MapControl` object.
```

> - Applying PolygonDisplayRules on locations that does not have Polygon geometry no longer causes a crash

Reviewer: ```suggestion
- Applying `PolygonDisplayRules` on Locations that does not have Polygon geometry no longer causes a crash
```

> - Fixed issue with tiles not loading, if a custom floor selector is implemented.

Reviewer: ```suggestion
- Issue with Tiles not loading, if a custom Floor Selector is implemented
```

> - Using near on MPQuery should now give a better ordering in the list in terms of what is near the set location.

Reviewer: ```suggestion
- Using `near` on `MPQuery` now gives a better ordering in the list in terms of what is near the set location
```
